HTML/index.html

```diff
@@ -3,44 +3,219 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Movie website Home Page</title>
-  <link rel="stylesheet" href="../CSS/index.css">
-
+  <title>Movie</title>
+  <link rel="stylesheet" href="../CSS/style.css">
+  <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
 </head>
 <body>
-  <!-- Header Section -->
```

Comment:
Very good 👍 on using comments to differentiate between sections you are working on.